packages/xrpl/src/models/ledger/XChainOwnedClaimID.ts

```diff
@@ -0,0 +1,73 @@
+import { XChainBridge } from '../common'
+
+import BaseLedgerEntry from './BaseLedgerEntry'
+
+/**
+ * An XChainOwnedClaimID object represents one cross-chain transfer of value
+ * and includes information of the account on the source chain that locks or
+ * burns the funds on the source chain.
+ *
+ * @category Ledger Entries
+ */
+export default interface XChainOwnedClaimID extends BaseLedgerEntry {
+  LedgerEntryType: 'XChainOwnedClaimID'
+
+  /** The account that checked out this unique claim ID value. */
+  Account: string
+
+  /**
+   * The door accounts and assets of the bridge this object correlates to.
+   */
+  XChainBridge: XChainBridge
+
+  /**
+   * The unique sequence number for a cross-chain transfer.
+   */
+  XChainClaimID: string
+
+  /**
+   * The account that must send the corresponding {@link XChainCommit} on the
+   * source chain. The destination may be specified in the {@link XChainCommit}
+   * transaction, which means that if the OtherChainSource isn't specified,
+   * another account can try to specify a different destination and steal the
+   * funds. This also allows tracking only a single set of signatures, since we
+   * know which account will send the {@link XChainCommit} transaction.
+   */
+  OtherChainSource: string
+
+  // TODO: type this better
```

Comment:
Should this TODO be resolved before merging, or can you create a follow up issue?

Reply:
(Same for rest of the TODOs for similar types)

Reply:
Good catch :)